.github/workflows/test-ocp-quickstart.yml

```
@@ -17,6 +17,13 @@ jobs:
         with:
           submodules: false
 
+      - name: Compute Submodule Cache Key
+        id: submodule-cache-key
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:3:22: Double quote to prevent globbing and word splitting

(shellcheck)